### PR TITLE
Consistently using `tx-time` and `tx-id` within http-server, console and http-client

### LINKED ISSUE #1230: HTTP API/Console consistently using `tx-time`/`tx-id` QPs throughout

### PR DESCRIPTION
**Pull request by danmason**
Resolves #1230

### CODE REVIEW

File: crux-http-client/src/crux/remote_api_client.clj [hunk 1]
```diff
@@ -157,14 +157,14 @@
                       :with-docs (:with-docs? opts)
 
                       :start-valid-time (some-> (:start-valid-time opts) (cio/format-rfc3339-date))
-                      :start-transaction-time (some-> (get-in opts [:start-tx ::tx/tx-time])
+                      :start-tx-time (some-> (get-in opts [:start-tx ::tx/tx-time])
```
Comment (jarohen) [inline crux-http-client/src/crux/remote_api_client.clj:160]: nit: needs a re-indent

**Comment by jarohen** [comment]
Thanks @danmason - one small formatting change and this is good to go

**Comment by danmason** [comment]
Reindented and ready to go :)

**Comment by jarohen** [comment]
Cheers!

**Comment by jarohen** [comment]
Cheers!

**Pull request merged**